superset/versions.py

```diff
@@ -14,24 +7,24 @@
         "authlib": "0.15.4",  # https://github.com/dpgaspar/Flask-AppBuilder/blob/v4.3.0/requirements-extra.txt#L10
     },
     {
-        "product": "3.0.1",
+        "product": "3.1.0",
         "python": "3.9",
         "vector": "0.39.0",
         "statsd_exporter": "0.26.1",
-        "authlib": "0.15.4",  # https://github.com/dpgaspar/Flask-AppBuilder/blob/v4.3.7/requirements-extra.txt#L7
+        "authlib": "1.2.1",  # https://github.com/dpgaspar/Flask-AppBuilder/blob/v4.3.10/requirements-extra.txt#L7
     },
     {
-        "product": "3.0.3",
+        "product": "3.1.3",
         "python": "3.9",
         "vector": "0.39.0",
         "statsd_exporter": "0.26.1",
-        "authlib": "1.2.1",  # https://github.com/dpgaspar/Flask-AppBuilder/blob/v4.3.10/requirements-extra.txt#L7
+        "authlib": "1.2.1",  # https://github.com/dpgaspar/Flask-AppBuilder/blob/release/4.4.1/requirements/extra.txt#L7
     },
     {
-        "product": "3.1.0",
+        "product": "4.0.2",
         "python": "3.9",
```

Comment:
question: Do we want to use Python 3.10 or even 3.11 here? It seems like Superset supports both newer versions.

Reply:
I think 3.9 to 3.11 is officially supported

Reply:
i can try 3.11 locally

Reply:
3.10 and 3.11 do not work out of the box. Would need adaptations of the docker file.

Reply:
We decided to tackle the Python bump at a later point in time. Merging as is.